com.unity.netcode.gameobjects/Tests/Runtime/NetworkVariableTests.cs

```diff
@@ -4,8 +4,63 @@
 using UnityEngine.TestTools;
 using NUnit.Framework;
 
+
+
 namespace Unity.Netcode.RuntimeTests
 {
+    public unsafe struct FixedStringStruct : INetworkSerializable
+    {
+        public fixed char FixedString[6];
```

Comment:
I would make this a `public unsafe char*`  instead of `public fixed char[]`. I think even with `fixed` this will cause a GC alloc... just a pinned one. With `char*` you can use `UnsafeUtility.Malloc`, which gives you freedom to support Temp and TempJob allocation schemes... just have to remember to `UnsafeUtility.Free` after is the only issue.